tcms/signals.py

```diff
@@ -175,3 +176,34 @@ def handle_comments_pre_delete(sender, **kwargs):
     instance = kwargs['instance']
 
     get_comments(instance).delete()
+
+
+def handle_emails_post_bug_save(sender, instance, created=False, **kwargs):
+    '''
+        Send email updates to assignee after they've been
+        assigned a bug on bug creation.
+    '''
+    if created is False or instance.assignee is None:
+        return
+
+    from django.conf import settings
+    from django.contrib.sites.models import Site
+
+    from tcms.core.utils.mailto import mailto
+
+    site = Site.objects.get(pk=settings.SITE_ID)
+    mailto(
+        template_name='email/post_bug_update/notify_new_assignee.txt',
```

Comment:
Use `post_bug_save` for diretory name to match existing convention and to match reality. 

Use `email.txt` for the file name for consistency with `post_case_delete` and `post_run_save` templates.